src/main/java/org/generationcp/middleware/manager/ontology/OntologyVariableDataManagerImpl.java

```diff
@@ -888,6 +910,14 @@ public boolean isVariableUsedInStudy(final int variableId) {
 		return !query.list().isEmpty();
 	}
 
+	public boolean isVariableUsedInGermplasm(final int variableId) {
```

Comment:
Related to current usage, I think we should abstract which entities a variable is used.  Thus we should not be exposing these as public:
isVariableUsedInGermplasm
isVariableUsedInBreedingMethods

As mentioned above, we can expose a public `hasUsage` method and the checking all relevant entities will be centralized there.

Reply:
done.